sktime/transformations/series/compose.py

```diff
@@ -96,3 +97,210 @@ def inverse_transform(self, Z, X=None):
         if not self.passthrough:
             z = self.transformer_.inverse_transform(z, X=None)
```

Comment:
I have just seen a silent issue in my code 😮 . Possibly, if we would have a transformer that uses `X` to transform `y`. Could you just fix it like this:
```suggestion
            z = self.transformer_.inverse_transform(z, X=X)
```
Thx! Dont want to open a PR for changing one symbol 😅